students/seunghun/users/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls import path
-from users.views import User_View
+from users.views import User_View, User_LoginView
 
 urlpatterns = [
-    path('', User_View.as_view()),
+    path('create', User_View.as_view()),
```

Comment:
POST method에 이미 create라는 의미가 포함되어있어 불필요합니다!
```suggestion
    path('signup', User_View.as_view()),
```
일반적으로 위와 같은 엔드포인트를 사용합니다.

Reply:
넵 ㅎ